content/en/tracing/trace_collection/library_config/java.md

```diff
@@ -253,6 +253,16 @@ When `false`, informational startup logging is disabled. Available for versions
 **Default**: `false`<br>
 When `true`, user principal is collected. Available for versions 0.61+.
 
+`dd.instrumentation.telemetry.enabled`
+: **Environment Variable**: `DD_INSTRUMENTATION_TELEMETRY_ENABLED`<br>
+**Default**: `true`<br>
+When `true`, the tracer collects [telemetry data][14]. Available for versions 0.104+. Defaults to `true` for versions 0.115+.
+
+`dd.telemetry.heartbeat.interval`
+: **Environment Variable**: `DD_TELEMETRY_HEARTBEAT_INTERVAL`<br>
+**Default**: `60`<br>
+Interval in seconds between telemetry messages to the backend. Available for versions 0.110+.
```

Comment:
This is a setting we use for testing, but customers shouldn't really set it. Is there a way to make hidden or something (and is it worth)?

Reply:
@paullegranddc I can exclude it from docs. There's quite a few flags that are meant for internal use only and are not present in the docs. So that should be ok.